gocourse03/main.go

```diff
@@ -0,0 +1,157 @@
+package main
+
+import (
+	"fmt"
+	"math/rand"
+	"time"
+)
+
+type (
+	Sector     string
+	rodentType string
+	FromTo     [2]Sector
+
+	Movement struct {
+		Time time.Time
+		FromTo
+		rodentId int
```

Comment:
Тут те саме правило, що й з `ID`:
```suggestion
		rodentID int
```

https://go.dev/wiki/CodeReviewComments#initialisms